crates/ruff/src/rules/flake8_pyi/rules/complex_assignment.rs

```diff
@@ -0,0 +1,45 @@
+use rustpython_parser::ast::{Expr, StmtAssign};
+
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks for assignments with multiple or non-name targets
+///
+/// ## Why is this bad?
+/// TODO
+///
+/// ## Example
+/// ```python
+/// a = b = int
+/// a.b = int
+/// ```
+///
+/// Use instead:
+/// ```python
+/// a = int
+/// b = int
+///
+/// TODO
+/// ```
+#[violation]
+pub struct ComplexAssignment;
+
+impl Violation for ComplexAssignment {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Stubs should not contain assignments with multiple targets or non-name targets.")
```

Comment:
I agree this message is a little tricky. I'm not sure users will understand what "non-name targets" means but I'm struggling to find a better message. Perhaps?

```suggestion
        format!("Stubs should not contain assignments to multiple targets or attributes.")
```

Reply:
or perhaps clearer to say "to attributes or multiple targets" to avoid confusion about "multiple attributes"